app/src/main/java/org/sopt/dosopttemplate/DoAndroidFragment.kt

```diff
@@ -0,0 +1,35 @@
+package org.sopt.dosopttemplate
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import org.sopt.dosopttemplate.databinding.FragmentHomeBinding
+
+class DoAndroidFragment: Fragment() {
+    private var _binding: FragmentHomeBinding? = null
+    private val binding: FragmentHomeBinding
+        get() = requireNotNull(_binding) { "바인딩 객체가 생성되지 않았다. 생성하고 불러라 임마!" }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentHomeBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        // 대부분의 로직은 여기에 구현합니다.
+        // 원두를 갈고~
+        // 커피를 내리고~
+    }
+
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
```

Comment:
```suggestion
        _binding = null
        super.onDestroyView()
```
이렇게 순서를 바꿔서 바인딩을 null 처리해주는 것이 더 안정성 있다고 합니다

Reply:
아하 이렇게 또 하나 배워갑니다 👍